bundles/org.openhab.binding.panasonicbdp/README.md

```diff
@@ -0,0 +1,243 @@
+# Panasonic Blu-ray Player Binding
+
+This binding connects Panasonic Blu-ray players from 2011/2012 and UHD Blu-ray players from 2018 to openHAB.  
+**Supported Blu-ray models:** DMP-BDT110, DMP-BDT210, DMP-BDT310, DMP-BDT120, DMP-BDT220, DMP-BDT320, DMP-BBT01 & DMP-BDT500.  
+**Supported UHD models:** DP-UB420/424, DP-UB820/824 & DP-UB9000/9004.  
+
+**Please Note:** The player must be on the same IP subnet as the openHAB server for this binding to function.
+If the connection to the player originates from a different subnet, 404 response errors are sent in response to all requests.
+
+To enable network remote control of the Blu-ray model players, configure the following settings:  
+**Player Settings/Network/Network Settings/Remote Device Settings**  
+Then make sure you have the following values set:  
+**Remote Device Operation: On**  
+**Registration Type: Automatic**  
+
+For the UHD models, Voice Control must be enabled for the player's http interface to be active:
+**Player Settings/Network/Voice Control: On**
+
+To enable the binding to control the player while off (network active while off), Quick Start mode must be On as follows:  
+**Player Settings/System/Quick Start: On**  
+
+**UHD Model Command Authentication:**  
+The UHD models require authentication to use the control API.
+A player key must be specified in the thing configuration in order for the `power`, `button` and `control` channels to work.
+UHD model players that are patched do not require a player key.
+See the online discussions of the DP-UB420/820/9000 players for more information.
```

Comment:
Where can these discussions be seen?

Reply:
https://www.avforums.com/threads/lets-try-again-to-put-the-free-in-regionfreedom.2441584/page-39#post-31906303
Not sure if it is desired to link there

Reply:
Maybe you can refer/link to avforums.com without a deep link? Just referring to "online discussions" seems a bit confusing since it could be anywhere in the Internet.

Reply:
I added a link to AVForums.